public/spec/features/resources_spec.rb

```diff
@@ -40,4 +40,26 @@
     )
     expect(page).to have_content(first_title)
   end
+
+  it 'should be able to see record identifiers in the tree of published resources, if configured' do
+    visit('/')
+    click_link 'Collections'
+    expect(current_path).to eq ('/repositories/resources')
+    click_link 'Published Resource'
+    if AppConfig[:pui_display_identifiers_in_resource_tree]
+      expect(page).to have_content('PUBLISHED_RESOURCE_ID: Published Resource')
+      expect(page).to have_content('PUBLISHED_AO_ID_0: Published Archival Object 0')
+      expect(page).to have_content('PUBLISHED_AO_ID_1: Published Archival Object 1')
+      expect(page).to have_content('PUBLISHED_AO_ID_2: Published Archival Object 2')
+      expect(page).to have_content('PUBLISHED_AO_ID_3: Published Archival Object 3')
+      expect(page).to have_content('PUBLISHED_AO_ID_4: Published Archival Object 4')
+    else
+      expect(page).not_to(have_content('PUBLISHED_RESOURCE_ID: Published Resource'))
+      expect(page).not_to(have_content('PUBLISHED_AO_ID_0: Published Archival Object 0'))
+      expect(page).not_to(have_content('PUBLISHED_AO_ID_1: Published Archival Object 1'))
+      expect(page).not_to(have_content('PUBLISHED_AO_ID_2: Published Archival Object 2'))
+      expect(page).not_to(have_content('PUBLISHED_AO_ID_3: Published Archival Object 3'))
+      expect(page).not_to(have_content('PUBLISHED_AO_ID_4: Published Archival Object 4'))
+    end
+  end
```

Comment:
@nbut1er I think, unfortunately, there is no way to do what you are trying to do here since the application is running in its own process, and we don't currently have a way to dynamically change its config settings. Perhaps we can look at adding a more dynamic `AppConfig` class down the road to make testing like this possible. For the present, I recommend just removing this test entirely as well as the one for the frontend.

Reply:
Thanks @quoideneuf, I've removed all of the new spec alterations